web/cypress/integration/consumer/create_and_delete_consumer.spec.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Delete Consumer', () => {
+        const root_name = `root_${new Date().valueOf()}`;
```

Comment:
what is `root_name` for?

Reply:
I can modify it, but please check the first e2e example, it is written like this, since it was apporved, I think the following e2e can be written like this.

Reply:
Just because the example is not appropriate, you cannot continue to imitate it. If you think it is incorrect here, change it to the correct one.

Please use camelCase in FE, and here should be consumerName or name.